nodebuilder/das/config.go

```diff
@@ -0,0 +1,36 @@
+package das
+
+import (
+	"fmt"
+
+	"github.com/celestiaorg/celestia-node/das"
+)
+
+// ErrMisConfig is an error that is returned by (*Config).Validate
+// when supplied with invalid values.
+var ErrMisConfig = fmt.Errorf("moddas misconfiguration")
```

Comment:
unnecessary error definition as it's returned in one place.

Reply:
I think it's a good idea though

Reply:
yup went with rene's suggestion. At second thought, I should have stuck to this style, because it would allow `nodebuilder` to check for `das.ErrMisConfig` (_`nodebuilder/das` that is_) which is more contextualized and clean than checking for `das.ErrInvalidOption`.

I think we can address this in a separate issue when we make sure we truly want that all proxy modules under `nodebuilder` become full proxies and leak nothing about their external origins.

Reply:
Tracking this in https://github.com/celestiaorg/celestia-node/issues/1289 
Can be closed if irrelevant